ツッコミメッセージを表示

glact1973 opened the pull request.

glact1973 committed:
ツッコミメッセージを表示

clonable-eden commented on server/router.js:
```diff
@@ -1,18 +1,27 @@
-  res.send('ツッコミタイム');
+  // 追加ここから
+  // ランダムメッセージを取得する
+  const inputPhrase = { phrase }.phrase;
```
この処理は不要な気がします！ここでやっていることは
```
{ phrase } -> "phrase"というキーに、phraseの値を持つオブジェクトを作っている
（別の書き方をすると `{ "phrase": phrase }` ですね）
.phrase -> "phrase"というキーの値を参照している
```
phraseという変数には、既に入力された文字列が入っているはずです！

glact1973 replied:
const {phrase}となっていたので、{}付きで扱うことしか考えてなかったです。
なるほど、そう読むのですね。
修正しました。

clonable-eden commented on server/router.js:
```diff
@@ -1,18 +1,27 @@
+  // ランダムメッセージを取得する
+  const inputPhrase = { phrase }.phrase;
+  const tukkomiMessage = getTukkomi();
+  if (inputPhrase === '')  res.send('入力してな'); 
```
`if (xxx) ... else { ... }` という書き方はおすすめしません。
両方ともつける or 両方ともつけない、どちらかに統一しましょう！

glact1973 replied:
了解です。
今回は両方つける方に統一しました

clonable-eden reviewed (commented):
@glact1973 機能的にはLGTMですが、コードの書き方についてコメントしました！

glact1973 committed:
再アップ

・if文の括弧"{}"をelseだけあり → 両方ありに揃えた
・入力文字の変数設定で重複していたので、変更した

glact1973 committed:
アップパスが間違っていたので削除

glact1973 committed:
再アップ

・if分の括弧{}をelseのあり → 両方ありに変更
・入力文字を変数化する'inputPrase'が不要だったため削除し、phraseを利用することとした

clonable-eden commented:
@glact1973 LGTMです！マージしましょう。
